tests/test_service.py

```diff
@@ -685,6 +685,19 @@ def test_rename_wishlist_product_invalid_content_type(self):
                                                             'product_name': 'surface_pro'},
                                                         headers={'content-type': 'text/plain'})
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+    
+    def test_remove_all_wishlist(self):
+        """ Test removing all items from Wishlist database """
+        wishlist1 = Wishlist(customer_id=1, name="name1")
+        wishlist1.save()
+        wishlist2 = Wishlist(customer_id=1, name="name2")
+        wishlist2.save()
+        created_wishlist_product = WishlistProduct(wishlist_id=wishlist1.id,
```

Comment:
`created_wishlist_product` hasn't been saved to the DB. So the product is not actually created. Can you take a look? Rest everything looks good.

Reply:
You are right! I will update